integration/harness_test.go

```diff
@@ -48,9 +49,13 @@ func run(t *testing.T, ftlConfigPath string, actions ...action) {
 	assert.NoError(t, err)
 
 	rootDir := internal.GitRoot("")
+	testData := filepath.Join(cwd, "testdata", "go")
 
 	if ftlConfigPath != "" {
-		t.Setenv("FTL_CONFIG", filepath.Join(tmpDir, ftlConfigPath))
+		tmpConfigPath := filepath.Join(tmpDir, ftlConfigPath)
```

Comment:
I think this will conflict with #1514 so you might want to sync with @deniseli

Reply:
Ah, thanks, I think @deniseli's is the better fix so I'll wait for that to land

Reply:
Mine's merged so you can rebase now. Sorry for the delay - I'm a bit in-and-out today sick. Thanks @wesbillman for merging it!